unfetter-discover/unfetter-ui
# Issue 640

supports unfetter-discover/unfetter#640

Added intelligent routing of assessments
Removed build:prod during docker build, because it is slow and needs lots of RAM.

#To Test
* pull branch
* `npm run checkup` verify everything looks good
* clean your mongo assessments `db.getCollection('stix').remove({ 'stix.type': 'x-unfetter-assessment' })`
* click on Apps -> Assessments 2.0
* verify you route to a create page
* create an assessment
* click Apps -> Assessment 2.0
* verify you land on your latest modified assessments summary page
